src/app/js/formats/text/ejs/EJSAdmin.js

```diff
@@ -0,0 +1,246 @@
+import PropTypes from 'prop-types';
+import { polyglot as polyglotPropTypes } from '../../../propTypes';
+import translate from 'redux-polyglot/translate';
+import RoutineParamsAdmin from '../../utils/components/admin/RoutineParamsAdmin';
+import React from 'react';
+import {
+    FormatDataParamsFieldSet,
+    FormatDefaultParamsFieldSet,
+} from '../../utils/components/field-set/FormatFieldSets';
+import updateAdminArgs from '../../utils/updateAdminArgs';
+import EJSEditor from './EJSEditor';
+import FormatFieldSetPreview from '../../utils/components/field-set/FormatFieldSetPreview';
+import { AllDataSets } from '../../utils/dataSet';
+import { EJSAdminView } from './EJSView';
+import FormatGroupedFieldSet from '../../utils/components/field-set/FormatGroupedFieldSet';
+
+export const defaultArgs = {
+    params: {
+        maxSize: 200,
+        orderBy: 'value/asc',
+    },
+    // Content of example.ejs
+    template: `<% /* Ceci est un exemple qui prendre en compte plusieurs types de donnée. */ %>
```

Comment:
<!-- TODO's are small, trivial, but necessary changes. Distinguishing todo comments from issues: or suggestions: helps direct the reader's attention to comments requiring more involvement.  -->
**todo:** Take fixed content from `example.ejs` back into this file‏